cluster/images/etcd/rollback/rollback.go

```diff
@@ -26,7 +26,10 @@ import (
 	"strings"
 	"time"
 
-	wal237 "k8s.io/kubernetes/third_party/forked/etcd237/wal"
+	// Uncomment when you want to rollback to 2.2.1 version.
+	oldwal "k8s.io/kubernetes/third_party/forked/etcd221/wal"
+	// Uncomment when you want to rollback to 2.3.7 version.
+	// oldwal "k8s.io/kubernetes/third_party/forked/etcd237/wal"
```

Comment:
But this will significantly complicate the code. Basically depending on the package, the
```
pkg.Create(walDir, metadata)
```
creates an object of a different type. So you need to have two completely separate branches of code for those two (in C++ we would be able to use template, but not in Go). I'm not sure it's worth it.